### PR TITLE
fix lint issues 

### PR DESCRIPTION
**Pull request by pavelpikta**
* fix lint issues
* fix swap
* fix condition 
* fix molecule tests

### CODE REVIEW

File: tasks/install-tf.yml [hunk 1]
```diff
@@ -89,44 +88,30 @@
       set_fact:
         silent_args: "{{ download_path }}/cd/Disk1/install/silent_response.rsp \
         ToolsAndFrameworks {{ base_root }}/endeca/ToolsAndFrameworks {{ workbench_password }}"
```
Comment (tgadiev) [inline tasks/install-tf.yml]: Would be better looking:
```yaml
      silent_args: >-
        {{ download_path }}/cd/Disk1/install/silent_response.rsp ToolsAndFrameworks
        {{ base_root }}/endeca/ToolsAndFrameworks {{ workbench_password }}
```

File: tasks/system/swap.yml [hunk 2]
```diff
@@ -26,9 +30,13 @@
 
     - name: "Make swap"
       command: "/sbin/mkswap {{ swapfile_path }}"
+      register: make_swap_result
+      changed_when: "make_swap_result.rc == 0"
```
Comment (tgadiev) [inline tasks/system/swap.yml]: Why quotes in conditional?

File: tasks/system/swap.yml [hunk 3]
```diff
@@ -26,9 +30,13 @@
...
 
     - name: "Swap on"
       command: "/sbin/swapon {{ swapfile_path }}"
+      register: swap_on_result
+      changed_when: "swap_on_result.rc == 0"
```
Comment (tgadiev) [inline tasks/system/swap.yml]: Quotes?

File: tasks/update-workbench-password.yml [hunk 4]
```diff
@@ -46,27 +46,23 @@
             status_code: "200"
             timeout: "300"
 
-      when: not (tf_version.split('.')[0] | int == 11 and tf_version.split('.')[1] | int <= 1)
+      when: tf_version is version ('11.1.0', '>')
```
Comment (tgadiev) [inline tasks/update-workbench-password.yml:49]: \> or < ?

File: tasks/update-workbench-password.yml [hunk 5]
```diff
@@ -46,27 +46,23 @@
...
     - name: "Set responses for manage_credentials.sh for versions higher 11.2"
       set_fact:
         credentials_responses:
           'Enter new password for user admin :': "{{ workbench_password }}"
           'Re-\enter new password to confirm :': "{{ workbench_password }}"
           'Do you want to replace it \[yes/no\] \?': "yes"
-      when: |
-        not (tf_version.split('.')[0] | int == 11 and
-        tf_version.split('.')[1] | int <= 2)
+      when: tf_version is version ('11.2.0', '>')
```
Comment (tgadiev) [inline tasks/update-workbench-password.yml:65]: \> ?

**Review by tgadiev — CHANGES_REQUESTED** [review]

File: tasks/update-workbench-password.yml (hunk 4 above)
Comment (pavelpikta) [reply tasks/update-workbench-password.yml:49]:
```yaml
- name: "Updating password in ifcr for versions higher 11.1"
```

File: tasks/update-workbench-password.yml (hunk 5 above)
Comment (pavelpikta) [reply tasks/update-workbench-password.yml:65]:
```yaml
- name: "Set responses for manage_credentials.sh for versions higher 11.2"
```

File: tasks/system/swap.yml (hunk 2 above)
Comment (pavelpikta) [reply tasks/system/swap.yml]: resolve in cd97303

File: tasks/system/swap.yml (hunk 3 above)
Comment (pavelpikta) [reply tasks/system/swap.yml]: resolve in cd97303

File: tasks/install-tf.yml (hunk 1 above)
Comment (pavelpikta) [reply tasks/install-tf.yml]: resolve in b2b3dac

**Review by tgadiev — APPROVED** [review]

**Pull request merged**